docker/puppetserver-standalone/docker-entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+chown -R puppet:puppet /etc/puppetlabs/puppet/ssl
+chown -R puppet:puppet /opt/puppetlabs/server/data/puppetserver/
+
+if test -n "${PUPPETDB_SERVER_URLS}" ; then
```

Comment:
Should this be part of the standalone?

Reply:
Yeah, I was wondering if the entrypoint should be overwritten by the puppetserver with puppetdb version? Not a blocker at all for this PR, just noticing that it seems to be a part of those concerns.

Reply:
I think I'd like to leave this as-is for now, this feels cleaner than duplicating the entrypoint script or having a new entrypoint script for puppetserver that adds this check then calls the existing puppetserver-standalone entrypoint.

Reply:
Executable scripts should not have a '.sh' extension.

See: https://stackoverflow.com/questions/27813563/what-is-the-bash-file-extension